README.md

```diff
@@ -65,6 +65,10 @@ View a list of other available targets with:
 $> vendor/bin/phing -l
 ```
 
+## Known Issues
```

Comment:
Can we remove this comment now that we're updating to PHP 8.2 on Circle?

Reply:
Yep! Done.